src/core/distributions/phylogenetics/substitution/PhyloCTMCClado.h

```diff
@@ -73,7 +73,7 @@ namespace RevBayesCore {
 
 
     private:
-        virtual void                                            simulate(const TopologyNode& node, std::vector< DiscreteTaxonData< charType > > &t, const std::vector<size_t> &perSiteRates);
+        virtual void                                            simulateClado(const TopologyNode& node, std::vector< DiscreteTaxonData< charType > > &t, const std::vector<size_t> &perSiteRates);
```

Comment:
Don't call this `simulate` since we already have a virtual function called `simulate`.